src/api/l_audio.c

```diff
@@ -305,20 +311,27 @@ int luaopen_lovr_audio(lua_State* L) {
 
   bool start = true;
   const char *spatializer = NULL;
+  int sampleRate = DEFAULT_SAMPLE_RATE;
   luax_pushconf(L);
   lua_getfield(L, -1, "audio");
   if (lua_istable(L, -1)) {
     lua_getfield(L, -1, "spatializer");
     spatializer = lua_tostring(L, -1);
     lua_pop(L, 1);
 
+    lua_getfield(L, -1, "sampleRate");
```

Comment:
For better or worse conf.lua fields use the Lua lowercase convention, so it should be `samplerate` for consistency